civictechprojects/static/css/partials/_EditProjectForm.scss

```diff
@@ -2,3 +2,8 @@
 .btn-background-project, .btn-background-project input[type="button" i] {
     background-color: $color-grey-medium;
 }
+
+.EditProjectForm-root .save_btn {
```

Comment:
This works to fix the symptom but I think we can address the root issue (a floated element floating outside its container) without it. Good news: We can just delete the float and fix the problem, since we align this button to the right twice. There's both a `pull-right` (float: right) and `text-right` (text-align: right) class applied to the button, so just remove pull-right and there's no need to relatively reposition because there's no more float.

Remove this CSS declaration entirely and then change EditProjectForm.jsx (line 369) and EditProfileController.jsx (line 245) from this:
`<div className="form-group pull-right">` 
to this:
`<div className="form-group">`

That fixes it without having to put more style rules on top.

Reply:
Thanks, your suggestion worked like a charm!